ASP.NET API/WAVC_WebApi/Controllers/FriendsManagerController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WAVC_WebApi.Data;
+using WAVC_WebApi.Models;
+
+namespace WAVC_WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [Authorize]
+    [ApiController]
+    public class FriendsManagerController : ControllerBase
+    {
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly ApplicationDbContext _dbContext;
+        private readonly FriendsManager _friendsManager;
+        private readonly int _searchResults = 10;
+
+        public FriendsManagerController(UserManager<ApplicationUser> userManager, ApplicationDbContext dbContext)
+        {
+            _userManager = userManager;
+            _dbContext = dbContext;
+            _friendsManager = new FriendsManager(dbContext);
+        }
+
+        [HttpGet]
+        [Route("Friends")]
```

Comment:
After renaming controller this line is obsolete. 
API call would look like `GET /api/Friends`  and that's enough